From 34efa4f9f4b9d5c8126edfa8e58de9abd7c3bed1 Mon Sep 17 00:00:00 2001
From: Mihails Strasuns <mihails.strasuns.contractor@sociomantic.com>
Date: Tue, 9 Feb 2016 20:37:37 +0200
Subject: [PATCH 01/14] Use sc-transitional.ini/dmd-transitional.conf

Looks for dmd configuration under different name from upstream compiler
---
 src/mars.d | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mars.d b/src/mars.d
index 17fbb7f59..641c732e1 100644
--- a/src/mars.d
+++ b/src/mars.d
@@ -395,11 +395,11 @@ private int tryMain(size_t argc, const(char)** argv)
     {
         version (Windows)
         {
-            global.inifilename = findConfFile(global.params.argv0, "sc.ini");
+            global.inifilename = findConfFile(global.params.argv0, "sc-transitional.ini");
         }
         else static if (__linux__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
         {
-            global.inifilename = findConfFile(global.params.argv0, "dmd.conf");
+            global.inifilename = findConfFile(global.params.argv0, "dmd-transitional.conf");
         }
         else
         {
-- 
2.13.2

